doc/api/stream.md

```diff
@@ -611,6 +621,17 @@ added:
 Number of times [`writable.uncork()`][stream-uncork] needs to be
 called in order to fully uncork the stream.
 
+##### `writable.writableErrored`
+
+<!-- YAML
+added:
+  REPLACEME
+-->
+
+* {Error}
```

Comment:
Just noting that it may not be an actual `Error` object since we can error with any value